### DISCUSSION TITLE
Find a photo by artist

### DISCUSSION THREAD

**Discussion by jean-louis67** [photoprism/photoprism#3472] [Q&A]
A priori, it is not possible to search with the _Artist_ filter. _Artist_ is a property displayed in the detail of a photo, but the corresponding search filter is not mentioned in the user guide.
I use the _Artist_ tag to designate the author of a photo and then I would like to search photos made by a given person. In the case of my photos, the term _Artist_ is a bit pretentious... but I haven't found a more suitable tag.
Thank you for your opinion!

**Comment by lastzero** [comment]
At the moment it is only indexed as part of the full text/keyword search. We had planned to combine it later with the camera information (serial number) in a special search filter. However, the Artist field is used in many different ways, so it can be a person's name, but sometimes it can be a company or a URL. Therefore, it is not always possible to easily normalize and match it, unlike, for example, camera names or subjects that you manually tag in a photo.

**Reply by jean-louis67** [reply]
Thank you very much for this answer
I fully understand these details and their context.
